code/modules/organs/internal/internal_organ_processes.dm

```diff
@@ -7,7 +7,7 @@
 /mob/living/carbon/human/proc/process_internal_ograns() //Calls all of the internal organ processes
 	if(should_have_process(OP_EYES))	//Bad way to do this, should be reworked to call procs from a list that's generated by the species.has_process list + extra processes gotten from organs (carrion)
 		eye_process()
-	if(should_have_process(OP_KIDNEYS))
+	if(should_have_process(OP_KIDNEY_LEFT) && should_have_process(OP_KIDNEY_RIGHT))
```

Comment:
In real life, humans can go fine with one kidney

Reply:
Usage of || is fine there

Reply:
Figgsed :DDDD